client/layout/logged-out.jsx

```diff
@@ -26,6 +23,11 @@ import { getSection, masterbarIsVisible } from 'state/ui/selectors';
 import BodySectionCssClass from './body-section-css-class';
 import GdprBanner from 'blocks/gdpr-banner';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
```

Comment:
I think this is the best place for this import, but I'd love to hear your opinion, @jsnajdr!

Reply:
Not bad :+1: 🙂

Reply:
Thank you! 🙂